packages/react-devtools-shared/src/devtools/views/SuspenseTab/SuspenseTab.js

```diff
@@ -1,8 +1,445 @@
+    iconType = state.inspectedElementHidden
+      ? 'panel-right-open'
+      : 'panel-right-close';
+  } else {
+    iconType = state.inspectedElementHidden
+      ? 'panel-bottom-open'
+      : 'panel-bottom-close';
+  }
+  return (
+    <Button
+      className={styles.ToggleInspectedElement}
+      data-orientation={orientation}
+      onClick={() =>
+        dispatch({
+          type: 'ACTION_SET_INSPECTED_ELEMENT_TOGGLE',
+          payload: null,
+        })
+      }
+      title={
+        state.inspectedElementHidden
+          ? 'Show Inspected Element'
+          : 'Hide Inspected Element'
+      }>
+      <ButtonIcon type={iconType} />
+    </Button>
+  );
+}
+
+function SuspenseTab(_: {}) {
+  const [state, dispatch] = useReducer<LayoutState, null, LayoutAction>(
+    layoutReducer,
+    null,
+    initLayoutState,
+  );
+
+  const wrapperTreeRef = useRef<null | HTMLElement>(null);
+  const resizeTreeRef = useRef<null | HTMLElement>(null);
+  const resizeTreeListRef = useRef<null | HTMLElement>(null);
+
+  // TODO: We'll show the recently inspected element in this tab when it should probably
+  // switch to the nearest Suspense boundary when we switch into this tab.
+
+  const {
+    inspectedElementHidden,
+    inspectedElementHorizontalFraction,
+    inspectedElementVerticalFraction,
+    treeListHidden,
+    treeListHorizontalFraction,
+  } = state;
+
+  useLayoutEffect(() => {
+    const wrapperElement = wrapperTreeRef.current;
+
+    setResizeCSSVariable(
+      wrapperElement,
+      'tree',
+      'horizontal',
+      inspectedElementHorizontalFraction * 100,
+    );
+    setResizeCSSVariable(
+      wrapperElement,
+      'tree',
+      'vertical',
+      inspectedElementVerticalFraction * 100,
+    );
+
+    const resizeTreeListElement = resizeTreeListRef.current;
+    setResizeCSSVariable(
+      resizeTreeListElement,
+      'tree-list',
+      'horizontal',
+      treeListHorizontalFraction * 100,
+    );
+  }, []);
+  useEffect(() => {
+    const timeoutID = setTimeout(() => {
+      localStorageSetItem(
+        LOCAL_STORAGE_KEY,
+        JSON.stringify({
+          inspectedElementHidden,
+          inspectedElementHorizontalFraction,
+          inspectedElementVerticalFraction,
+          treeListHidden,
+          treeListHorizontalFraction,
+        }),
+      );
+    }, 500);
+
+    return () => clearTimeout(timeoutID);
+  }, [
+    inspectedElementHidden,
+    inspectedElementHorizontalFraction,
+    inspectedElementVerticalFraction,
+    treeListHidden,
+    treeListHorizontalFraction,
+  ]);
+
+  const onResizeStart = (event: SyntheticPointerEvent<HTMLElement>) => {
+    const element = event.currentTarget;
+    element.setPointerCapture(event.pointerId);
+  };
+
+  const onResizeEnd = (event: SyntheticPointerEvent<HTMLElement>) => {
+    const element = event.currentTarget;
+    element.releasePointerCapture(event.pointerId);
+  };
+
+  const onResizeTree = (event: SyntheticPointerEvent<HTMLElement>) => {
```

Comment:
There's probably an abstraction here that can be shared with the Components tab. I had to slightly adjust where and what percentage values with set though so I opted for duplicating. At least during experimentation phase.